setup.cfg

```diff
@@ -48,15 +110,3 @@ exclude_lines =
     raise
     if 0:
     if __name__ == .__main__.:
-
-[isort]
```

Comment:
Why do you remove this ?

Reply:
I think this config is duplicated, the isort use configs from `pyproject.toml` by default